### ISSUE TITLE
Add functional to pass client certs

### ISSUE THREAD

**Issue by bscheshirwork** [yiisoft/yii2-httpclient#119]
### What steps will reproduce the problem?
I want to pass client cert [like this](https://github.com/bscheshirwork/docker-yii2-app-advanced-rbac/blob/api/docs/example-api-ssl-client.md):
stream
```php
            'ssl' => [
                'verify_peer' => true,
                'allow_self_signed' => true,
                'cafile' => Yii::$app->basePath . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'ca.crt',
                'local_cert' => Yii::$app->basePath . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'client01.crt',
                'local_pk' => Yii::$app->basePath . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'client01.key',
                'passphrase' => 'client01 crt example pass phrase',
            ],
```
curl
```php
curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, 2);
curl_setopt($ch, CURLOPT_CAINFO, Yii::$app->basePath . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'ca.crt');
curl_setopt($ch, CURLOPT_SSLKEY, Yii::$app->basePath . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'client01.key'); 
curl_setopt($ch, CURLOPT_SSLCERT, Yii::$app->basePath . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'client01.crt');
curl_setopt($ch, CURLOPT_SSLCERTPASSWD, 'client01 crt example pass phrase');
```
### What's expected?
options `sslLocalCert`, `sslLocalPk`, `sslPassphrase`

### What do you get instead?
not implemented

### Additional info

| Q                       | A
| ----------------------- | ---
| Yii version             | dev 
| Yii HTTP Client version | dev
| PHP version             |7.1.10
| Operating system        |alpine

**Comment by bscheshirwork** [comment]
I prepare PR for it...

**Comment by klimov-paul** [comment]
Resolved by commit 6d47b2f0e83462a7ce0abcece6a80867cb6c7021
